.travis.yml

```
@@ -1,6 +1,10 @@
 language: go
 
 go:
-  - 1.8
-  - 1.7
-  - 1.6
\ No newline at end of file
+  - tip
+  - 1.15.x
+  - 1.14.x
+  - 1.13.x
+
```

Let's at least support back to 1.12.  I mean, clearly this thing should work in whatever versions we like since go is good like that with forwards compatibility. 

```suggestion
  - 1.13.x
  - 1.12.x
  
```